ccp/modules/mtba-setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Consider using flags -e and/or -u

Reply:
It is exactly like the current "id-management-setup.sh". When we implement a solution, I would do it uniformly for all modules. Do we change it everywhere like this?

Reply:
Fixed: #!/bin/bash -e